[Bug?]: Type AuthProvider no longer exported from `@redwood/auth`

### What's not working?

![grafik](https://github.com/user-attachments/assets/9f8ee84f-b478-4638-95ae-470504fd06eb)

> Cannot find module '@redwoodjs/auth/dist/AuthProvider/AuthProvider' or its corresponding type declarations.ts(2307)

### What's your environment? (If it applies)

```shell
OS: Linux 6.12 Manjaro Linux
    Shell: 5.2.37 - /bin/bash
  Binaries:
    Node: 20.16.0 - /tmp/xfs-06ce3f05/node
    Yarn: 4.4.0 - /tmp/xfs-06ce3f05/yarn
  Databases:
    SQLite: 3.47.1 - /usr/bin/sqlite3
  npmPackages:
    @redwoodjs/auth-azure-active-directory-setup: 8.4.1 => 8.4.1 
    @redwoodjs/cli-storybook-vite: 8.4.1 => 8.4.1 
    @redwoodjs/core: 8.4.1 => 8.4.1 
    @redwoodjs/project-config: 8.4.1 => 8.4.1
```


### Are you interested in working on this?

- [X] I'm interested in working on this

--- a/.changesets/11766.md
+++ b/.changesets/11766.md
@@ -1,0 +1,3 @@
+- Fix AuthProviderProps export missing from `@redwood/auth` (#11766) by @Philzen
+
+`AuthProviderProps` is now exported from `@redwood/auth`

--- a/packages/auth/src/index.ts
+++ b/packages/auth/src/index.ts
@@ -1,4 +1,5 @@
 export type { AuthContextInterface, CurrentUser } from './AuthContext.js'
+export type { AuthProviderProps } from './AuthProvider/AuthProvider.js';
 export { useNoAuth } from './useAuth.js'
 export type { UseAuth } from './useAuth.js'
 export { createAuthentication } from './authFactory.js'

